src/components/calculator.js

```diff
@@ -0,0 +1,119 @@
+/* eslint-disable no-unused-vars */
```

Comment:
- [ ] I noticed that you disabled **Eslint** . Please remove the **eslint disable** statement and try to fix all the errors you are getting. Because ` Eslint` helps developers to write clean JavaScript codes and adapt best practices when writing the code.  :+1:

Reply:
Please suggest a way in which these errors can be removed I have tried all means possible

Reply:
- [ ] Please refer to my comment in `App.js` on line `1` to fix this issue 👍